src/app/tasks/project-tasks-list/project-tasks-list.component.scss

```diff
@@ -0,0 +1,73 @@
+@import '../../../../build/assets/doubtfire.css';
+
+.project-tasks-list {
```

Comment:
Use Tailwind CSS for all the newer components

Reply:
Just made changes using tailwind in line style and tailwind.config.js